### PR TITLE
define HttpVersion.Version30 and use where appropriate

### PR DESCRIPTION
**Pull request by geoffkizer**
Contributes to #1293

@scalablecory @dotnet/ncl

### CODE REVIEW

File: src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs [hunk 1]
```diff
@@ -9,5 +9,6 @@ public static class HttpVersion
         public static readonly Version Version10 = new Version(1, 0);
         public static readonly Version Version11 = new Version(1, 1);
         public static readonly Version Version20 = new Version(2, 0);
+        public static readonly Version Version30 = new Version(3, 0);
```
Comment (stephentoub) [inline src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs:12]:
```suggestion
        /// <summary>Defines a Version instance for HTTP 3.0.</summary>
        public static readonly Version Version30 = new Version(3, 0);
```

Comment (geoffkizer) [reply src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs:12]: May as well do the others too, no?

Comment (stephentoub) [reply src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs:17]: Yup, GitHub just doesn't let me make suggestions on them :)

Comment (geoffkizer) [reply src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs:17]: I never know how to think about whether to add doc comments to existing APIs that don't have them.

On the one hand, if the API already has good documentation, I don't want to arbitrarily add some garbage that's worse than what already exists, and end up making things worse.

On the other hand, sometimes you end up with this: https://docs.microsoft.com/en-us/previous-versions/visualstudio/hh137994(v=vs.118)

@carlossanlop Guidance here?

**Review by stephentoub — APPROVED** [review]

**Pull request merged**